ionos_sbx/marketplace/access-node/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: access-node
 description: Chart holder for ArgoCD
 type: application
```

Comment:
Please prefix commit message with env name

Reply:
Please also do the rebase

Reply:
Done. Is there a documentation for those "things"?

Reply:
Is it really you, Stefan?

Reply:
yes

Reply:
Welcome back, Stefan. As for "Done. Is there a documentation for those "things"?" - there are not many changes to document since July. We only added requirement to prefix the commit wiith the env name and request to squash and rebase changes (and resolve conflicts) before the merge.